src/formio/components/productPrice.ts

```diff
@@ -0,0 +1,16 @@
+import {InputComponentSchema} from '..';
+
+type Validator = 'required';
+type TranslatableKeys = 'label' | 'description' | 'tooltip';
+
+export type ProductPriceInputSchema = InputComponentSchema<never, Validator, TranslatableKeys>;
+
+/**
+ * @group Form.io components
+ * @category Base types
+ */
```

Comment:
I think it'd be good to add some documentation here to explicitly state this is an experimental/unstable feature that may be changed at anytime without prior notice :)